docs/theory/heat_physics.rst

```diff
@@ -0,0 +1,372 @@
+
+Node
+~~~~
+
+The node conserves the flow with :eq:`eq:flow_balance` and the energy with :eq:`eq:general_energy_balance`.
+
+All heads connected to the node must be equal to ensure hydraulically feasible solution:
+
+.. math::
+    :label: eq:node_head
+
+    H_i = H^a \;\; \forall i \in I^a \;\; a \in A_{nodes},
+
+where :math:`A_{nodes}` is the set of all nodes and :math:`H^a` is the headloss for node :math:`a`.
+
+Asset Physics
+-------------
+
+Source
+~~~~~~
+
+The source adds thermal power to the network.
+The energy balance is given by :eq:`eq:general_energy_balance` where :math:`\dot{Q}^a_{consumed}` is equal to the (negative) value of the produced heat.
+
+The addition of energy is executed by increasing the temperature of the incoming water (the return network) to the outgoing supply temperature.
+Therefore, the volumetric flow rate is linked to the outgoing thermal power with the outgoing supply temperature with equality constraints:
+
+
+.. math::
+    :label: eq:source_heat2discharge
+
+    \sum_{i \in I^a_{out}} \dot{Q}^a_i = c_p \rho \dot{V}^a T_{sup} \;\; \forall a \in A_{prod},
+
+where :math:`A_{prod}` is the set of all producers.
+
+The equality constraints ensure that the thermal power equals what can be physically transported by the flow.
+No constraints are set on the returning flow at the in port, as this is already implied by the constraints on pipes.
+
+The modelling errors from not modelling temperature are present in various parts of the costs.
+The heat losses will be overestimated giving an overestimate w.r.t. the cost of the thermal energy produced.
+The overestimation of the thermal power required will induce an overestimation of the flow and thereby of the transportation and pipe sizing.
+
+The source is assumed to act within one hydraulically coupled network, :eq:`eq:flow_balance`.
+
+A source is modelled with a pump to reach its desired flow rate and head:
+
+.. math::
+    :label: eq:source_pump_dh
+
+    H^a_{in} + dH_{pump} = H^a_{out} \;\; \forall a \in A_{prod}.
+
+Demand
+~~~~~~
+
+A demand extracts thermal power from the network, defined by :eq:`eq:general_energy_balance` where :math:`\dot{Q}^a_{consumed}` is the consumed thermal power.
+
+Similar to the source, no constraints are required at the in-going supply side of the demand, and
+an equality constraint relates outgoing thermal power with volumetric flow:
+
+.. math::
+    :label: eq:consumer_heat2discharge
+
+    \sum_{i \in I^a_{out}} \dot{Q}^a_i = c_p \rho \dot{V}^a T_{ret} \;\; \forall a \in A_{demand},
+
+where :math:`A_{demand}` is the set of demand assets.
+
+The combination of constraints ate the producers and demands, results in a smaller achieved temperature difference at the demand than the difference between the given temperatures for the supply and return side, and a larger achieved temperature difference at the producer.
```

Comment:
potentially add at the end of the sentence, "due to pipe heat losses".

Reply:
Has been added